contracts/transfer/circuits/src/dusk_contract/send_to_contract_obfuscated.rs

```diff
@@ -143,6 +171,22 @@ mod tests {
     use super::*;
     use anyhow::Result;
     use dusk_plonk::commitment_scheme::kzg10::PublicParameters;
+    use poseidon252::sponge::sponge::sponge_hash;
+
+    fn schnorr_sign(
+        sk: JubJubScalar,
+        message: BlsScalar,
+    ) -> (JubJubScalar, AffinePoint, AffinePoint) {
+        let pk = AffinePoint::from(GENERATOR_NUMS_EXTENDED * sk);
+        let r = JubJubScalar::random(&mut rand::thread_rng());
+        let R = AffinePoint::from(GENERATOR_EXTENDED * r);
+        let h = sponge_hash(&[message]);
+        let c_hash = sponge_hash(&[R.get_x(), R.get_y(), h]);
+        let c_hash = c_hash & BlsScalar::pow_of_2(250).sub(&BlsScalar::one());
+        let c = JubJubScalar::from_bytes(&c_hash.to_bytes()).unwrap();
+        let U = r - (c * sk);
+        (U, R, pk)
+    }
```

Comment:
Is this function used to be able to test the Schnorr sigs? If that's the case, please add some comments since it shouldn't be here otherways.

Reply:
okay